command/git.go

```diff
@@ -137,13 +137,24 @@ func (c *gitFetchProject) buildCloneCommand(conf *model.TaskConfig) ([]string, e
 
 	if conf.GithubPatchData.PRNumber != 0 {
 		branchName := fmt.Sprintf("evg-pr-test-%s", util.RandomString())
-
-		gitCommands = append(gitCommands, []string{
+		var ref, commitToTest string
+		if conf.Task.Requester == evergreen.MergeTestRequester {
```

Comment:
The new requester type indicates to the get_project command to check out the merge commit

Reply:
does it make more sense to make this a method on Task?

Reply:
I think I was able to do it.